openapi.yaml

```diff
@@ -292,46 +284,68 @@ paths:
               schema:
                 $ref: "#/components/schemas/PassportResponse"
         "400":
-          description: "Invalid name supplied"
+          description: "Invalid data"
         "404":
           description: "Passport not found"
 
-  /lostPassport:
```

Comment:
udated endpoints and was added new requets type with one single value to send which is false to deactivate a passport, and answer should be a recomendation to create a new one